packages/react/src/components/Authenticator/ConfirmSignUp/ConfirmSignUp.tsx

```diff
@@ -60,14 +61,15 @@ export function ConfirmSignUp() {
       onSubmit={handleSubmit}
     >
       <fieldset
-        style={{ display: 'flex', flexDirection: 'column' }}
-        className="amplify-flex"
+        className={classNames('amplify-flex', 'amplify-authenticator__column')}
```

Comment:
These have all been switched to use the `Flex` component but this isn't an option for the angular and vue implementations.  It seems to make more sense to have this class be `amplify-flex--column` but would introducing that class set the expectation that we would at the very least create `amplify-flex--row` and at the most create CSS classes for all of our primitive style prop options?  I went with `amplify-authenticator__column` to avoid opening that can of worms.